### PR TITLE
Bump Rails from 5.0.0.1 to 5.0.1

### PR DESCRIPTION
**Pull request by aliuk2012**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -226,35 +224,35 @@ GEM
...
-      activesupport (= 5.0.0.1)
+    railties (5.0.1)
+      actionpack (= 5.0.1)
+      activesupport (= 5.0.1)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
     rainbow (2.1.0)
-    rake (11.2.2)
+    rake (12.0.0)
```
Comment (misaka) [inline Gemfile.lock]: Interesting ... that's a big bump for `rake`. I guess we tested our rake tasks?

Comment (aliuk2012) [inline Gemfile.lock]: @misaka We don't have any rake tasks to test ;-)

Comment (aliuk2012) [inline Gemfile.lock]: @misaka could you review?

**Comment by EdwardAndress** [comment]
@aliuk2012 - I think we can merge this one, given that @misaka has seen it and only queried the rake task (and it's a small change).

**Comment by EdwardAndress** [comment]
... and that he's probably not able to do another review today.

**Pull request merged**